ChangeLog

```diff
@@ -0,0 +1,134 @@
+CHANGES
```

Comment:
This duplicates HISTORY.rst, I don't think this file should have been added.

Maybe these two files could be added to .gitignore instead to avoid that kind of confusion in the future.

Reply:
Good idea! Best to prevent this for the future.